### ISSUE TITLE
Work Description allows text to extend beyond visual borders of field.

### ISSUE THREAD

**Issue by spamspambaconspam** [internetarchive/openlibrary#3611]
While entering data in the WORK DESCRIPTION field:

If lines are numbered, the numbering itself is set outside the border of the field, as if the string is left-aligned by the first alphanum char following the line numbering string. 

I can play around with it and get it to stay within the borders, but ONLY if I add a trailing zero.

It seems to be hyper-responsive to typing the digit "1" and a period/decimal char.
Zeroes, twos, etc don't seem to create the same response. 

First screenshot is of my numbered list, just as I entered it, with the results previewed.
Second screenshot adds a trailing zero to enforce the constraints.
Third s/shot shows how aggressive it is - it's already numbering the line, just with a zero and decimal.
**Fourth s/shot is also kind funky** --  I removed the line numbering from my line #2, and it did the following:
it concatenated both lines #1 & #2, and then continued numbering from what WAS line #3. 
The line numbering you see in the edit box is MINE, from my plain text paste.  It was not generated via markdown.
So it actually removed part of my original string. 
 
I also added a few more screenshots - I was playing around to see what exact sequence of chars would "break" the formatting. 
They may help you as clues.
    

### Evidence / Screenshot (if possible)
![image](https://user-images.githubusercontent.com/9367321/88114021-fe3ec380-cb80-11ea-8fa7-3e9afb061ec9.png)
![image](https://user-images.githubusercontent.com/9367321/88114210-5d9cd380-cb81-11ea-837d-e9585079d6a6.png)
![image](https://user-images.githubusercontent.com/9367321/88114915-f4b65b00-cb82-11ea-9286-23dedb31f702.png)
![image](https://user-images.githubusercontent.com/9367321/88114699-78bc1300-cb82-11ea-80f0-5333df972364.png)
![image](https://user-images.githubusercontent.com/9367321/88115021-2deecb00-cb83-11ea-9c44-195b9b16ea4e.png)
![image](https://user-images.githubusercontent.com/9367321/88115043-39da8d00-cb83-11ea-8364-a41af251ab15.png)
![image](https://user-images.githubusercontent.com/9367321/88115065-4959d600-cb83-11ea-89b6-8afcea7144d2.png)

### Relevant url?
This is where I noticed the issue.
I've left the numbering in place, in case you want to see it in prod without having to jump to prod :)

https://openlibrary.org/books/OL23764632M/The_Best_Short_Stories_of_1915

### Steps to Reproduce
Start numbering at one.

### Details

- **Logged in (Y/N)?** Yessir
- **Browser type/version?** Edge
- **Operating system?** Win 10
- **Environment (prod/dev/local)?** prod

**Comment by nynaalekhya** [comment]
@jdlrobson  Can I work on this issue?

**Comment by jdlrobson** [comment]
@nynaalekhya I think focusing on https://github.com/internetarchive/openlibrary/issues/3713 would be better. This doesn't look super high priority to me and the description could do with some additional data.

@spamspambaconspam  could you provide some more information here such as the URI you were using to generate this bug report? I'm  unable to replicate it..

**Comment by nynaalekhya** [comment]
@jdlrobson  Thank you for assigning me the other issues.
But since the issues have already been worked on by someone else, can I take up this one?

**Comment by nynaalekhya** [comment]
Maybe this could summarized as,

Go to https://openlibrary.org/books/OL23764632M/The_Best_Short_Stories_of_1915 > Click on "work details" tab

1st case:  The preview text overflows when lengthy sentence is entered 

![cap3](https://user-images.githubusercontent.com/50094420/103927502-4d48c880-50cf-11eb-8ad2-160065b58347.JPG)

**Comment by jdlrobson** [comment]
@nynaalekhya I don't recommend this particular task as I don't actually understand the issue here and want further information. How about #2597 ?

**Comment by nynaalekhya** [comment]
2nd case:  Enter 01. and enter spacebar twice  in "How would you describe book" textbox to reproduce the following,

Input:

![cap1](https://user-images.githubusercontent.com/50094420/103927686-9dc02600-50cf-11eb-8b58-18d1a0baaf4d.JPG)

Output:

![cap2](https://user-images.githubusercontent.com/50094420/103927734-b0d2f600-50cf-11eb-8fe6-78f8dc1ccfbe.JPG)

**Comment by nynaalekhya** [comment]
> @nynaalekhya I don't recommend this particular task as I don't actually understand the issue here and want further information. How about #2597 ?

@jdlrobson 
Yeah sure, I will take up that issue.
Thank you

**Comment by nynaalekhya** [comment]
@jdlrobson #2597 Is this closed?

**Comment by jdlrobson** [comment]
Yep I think any of these issues would be better, but I've not been checking them in a while, so any help working out which are still broken would also be helpful :) https://github.com/internetarchive/openlibrary/issues?q=is%3Aopen+label%3A%22Lead%3A+%40jdlrobson%22+label%3A%22Priority%3A+3%22+-label%3A%22Type%3A+Feature+Request%22

**Comment by nynaalekhya** [comment]
@jdlrobson 
Certainly! Thank you.

**Comment by RayBB** [comment]
I also don't see clarity on this one and think it's more likely we'll replace the editor before we fix all the small formatting weirdness so I'm going to close this.

Please feel free to open a new issue and follow the issue template if you find this still to be a problem.
